crates/cargo-test-support/src/registry.rs

```diff
@@ -214,6 +229,23 @@ pub fn init() {
     fs::create_dir_all(alt_api_path().join("api/v1/crates")).unwrap();
 }
 
+pub fn init_alt2_registry() {
```

Comment:
Can this be written so that the other two registries reuse the same code instead of duplicating it?

Reply:
I replaced this with a new function `init_registry` and the existing codes for creating registry call the new function. I am a little worried that there are many arguments for this new function, but I couldn't come up with another good way to write it.